roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
```

Comment:
Yeah this is a much bigger issue for sure - ultimately we need to be config driven (kube api objects, not layers on top) and then be able to cleanly layer the customizations we need in Ansible.  That really has to start with a set of Kube API config objects (maybe 2 or 3 sets, depending on how we want to break out policy / secrets from those objects).

Recreating Kube config by hand as ansible is going to rapidly become unmaintainable.  Kube will get better at running raw config, and so there will be gravity back to raw config objects that will be difficult and costly to reproduce elsewhere.  We need to be on a track where the core config is copied into a playbook and then further customized before being applied.

Let's set some time up to talk this week, but I think we have enough customizations with the yaml editor module today that this should be "import the upstream template", "parameterize a few things", and "deploy".